### ISSUE TITLE
Mask output of Load Images For Loop is (64, 64) instead of (1, 64, 64) — missing batch dimension?

### ISSUE THREAD

**Issue by king-kero** [yolain/ComfyUI-Easy-Use#1027]
Thank you always.
When migrating a workflow that normally uses the standard Load Image node to a loop-based batch process, I replaced the mask input with the mask output of Load Images For Loop. This caused downstream processing (SeedVR2's tiled VAE encode) to hang / appear frozen.

Using easy-use's Show Tensor Shape node, I confirmed the shapes:

Standard Load Image → mask shape (1, 64, 64)
Load Images For Loop → mask shape (64, 64) (batch dimension missing)

Looking at the source, the standard ComfyUI LoadImage node calls mask.unsqueeze(0) before returning the mask (both for the alpha-derived mask and the default all-zero fallback), but loadImagesForLoop in py/nodes/image.py does not:

python
if 'A' in i.getbands():
    mask = np.array(i.getchannel('A')).astype(np.float32) / 255.0
    mask = 1. - torch.from_numpy(mask)
else:
    mask = torch.zeros((64, 64), dtype=torch.float32, device="cpu")

Is this missing batch dimension intentional, or is it an oversight? If unintentional, could mask be returned as (1, H, W) (matching the standard LoadImage output) instead of (H, W), e.g. by adding .unsqueeze(0) before returning?
